package.json

```diff
@@ -150,8 +149,8 @@
     "workbox-webpack-plugin": "7.0.0"
   },
   "engines": {
-    "yarn": ">= 1.0.0",
-    "node": ">=16"
+    "yarn": ">= 4.1.1",
+    "node": ">=18.16.0"
```

Comment:
still not sure what happens with this runs on our *unfortunately* still node 16 EB instance

Reply:
I don't think it's a problem, since the aws never runs anything else than expressjs. 
All the rest is just for CI and local dev, we will be fine I believe